lib/cosmos/tools/test_runner/test_runner.rb

```diff
@@ -822,6 +822,7 @@ def show_select
                                              Qt::DialogButtonBox::Cancel)
         connect(button_box, SIGNAL('rejected()'), box, SLOT('reject()'))
         connect(button_box, SIGNAL('accepted()')) do
+          ScriptRunnerFrame.instance = @script_runner_frame
```

Comment:
To be honest I'm not exactly sure why this is necessary. I see it being done in other places. I thought it should be done once at initialization and be set for the duration.